### PR TITLE
Style updates to improve SPDF compliance

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Updates meta data and file naming conventions to improve compatibility with SPDF systems.

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Inspection of files

### CODE REVIEW

**Comment by jklenzing** [comment]
TODO:
- [ ] add download support to get all files
- [ ] Update val_max of nan with statistically derived values

File: ops_reach/instruments/methods/reach.py [hunk 1]
```diff
@@ -387,3 +451,62 @@ def generate_metadata(header_data):
         meta.labels.fill_val: -999}
 
     return meta
+
+
+def download(date_array, tag, inst_id, data_path=None, **kwargs):
+    """Download data (placeholder). Doesn't do anything.
```
Comment (jklenzing) [inline ops_reach/instruments/methods/reach.py:457]:
```suggestion
    """Download data from zenodo.
```

File: ops_reach/instruments/aero_reach.py [hunk 2]
```diff
@@ -8,21 +8,19 @@
 name
     'reach'
 inst_id
-    '101', '105'
+    '101', '105', '113', '133', '135'
```
Comment (jklenzing) [inline ops_reach/instruments/aero_reach.py:11]: update!

File: ops_reach/instruments/methods/reach.py [hunk 3]
```diff
@@ -259,15 +328,24 @@ def generate_metadata(header_data):
...
                               meta.labels.units: '#/cm^2/sr/s',
                               meta.labels.min_val: 0.0,
-                              meta.labels.max_val: np.nan}
+                              meta.labels.max_val: 1.0e9,
+                              'SCALEMIN': 1.0,
+                              'SCALEMAX': 1.0e9,
+                              'SCALETYP': 'log'}
     meta['species1'] = {meta.labels.name: 'Most probable species',
                         meta.labels.notes: '; '.join(('0 (not currently used)',
                                                       '1 – protons, 2-electrons',
```
Comment (JonathonMSmith) [inline ops_reach/instruments/methods/reach.py:351]:
```suggestion
                                                      '1-protons, 2-electrons',
```

File: ops_reach/instruments/methods/reach.py [hunk 4]
```diff
@@ -279,15 +357,24 @@ def generate_metadata(header_data):
...
                               meta.labels.units: '#/cm^2/sr/s',
                               meta.labels.min_val: 0.0,
-                              meta.labels.max_val: np.nan}
+                              meta.labels.max_val: 1.0e11,
+                              'SCALEMIN': 1.0,
+                              'SCALEMAX': 1.0e11,
+                              'SCALETYP': 'log'}
     meta['species2'] = {meta.labels.name: 'Most probable species',
                         meta.labels.notes: '; '.join(('0 (not currently used)',
                                                       '1 – protons, 2-electrons',
```
Comment (JonathonMSmith) [inline ops_reach/instruments/methods/reach.py:380]:
```suggestion
                                                      '1-protons, 2-electrons',
```

**Review by JonathonMSmith — CHANGES_REQUESTED** [review]
these two hyphens are unicode, so we should change them

**Pull request merged**